src/index.ts

```diff
@@ -1,15 +1,31 @@
-export * from './account'
-export * from './amount'
+// TODO review exports
 export * from './app'
 export * from './app-client'
 export * from './app-deploy'
 export * from './asset'
-export * from './config'
 export * from './debugging'
-export * from './dispenser-client'
 export * from './indexer-lookup'
-export * from './localnet'
-export * from './network-client'
 export * from './transaction'
 export * from './transfer'
+
+// Legacy exports - remove after 2 x major version bumps
+export * from './account'
+export * from './dispenser-client'
+export * from './localnet'
+export * from './network-client'
+
+// Up to date exports
+export * from './amount'
+export * from './config'
 export * from './types/algorand-client'
+
+// Default export with intellisense of the most pertinent parts of AlgoKit Utils
+import * as amount from './amount'
+import * as config from './config'
+import * as algorand from './types/algorand-client'
+
+export default {
+  ...amount,
+  ...config,
+  AlgorandClient: algorand.AlgorandClient,
+}
```

Comment:
Would be great to get feedback about this. I think it's a good idea to make a clean break against the default way you import algokit utils and get baseline intellisense. This should end up being a very small set of things since most things will come from AlgorandClient.

It results in a cleaner import of `import algokit from "package"` rather than `import * as algokit from "package"` and means you can quickly tell with a project wide search if you are using new or old way of using algokit utils across a project and gradually migrate.

Reply:
What about:

`import { AlgorandClient, Config } from 'algokit-utils/core'`

Reply:
or we just avoid this and people deal with the noise and do:

`import { AlgorandClient, Config } from '@algorandfoundation/algokit-utils'`